### PR TITLE
cmd/server: user overrides for Unsafe or Exceptions on SDNs

### PR DESCRIPTION
**Pull request by adamdecaf**
Issue: https://github.com/moov-io/ofac/issues/37

### CODE REVIEW

File: cmd/server/customers.go [hunk 1]
```diff
@@ -170,27 +250,46 @@ type customerStatusRequest struct {
...
 		}
+
 		req.Status = strings.ToLower(strings.TrimSpace(req.Status))
-		switch req.Status {
-		case "blocked":
-			// update customer status to blocked
-		default:
-			if req.Status == "" {
-				moovhttp.Problem(w, errors.New("no status provided"))
+		if req.Status == "blocked" {
```
Comment (adamdecaf) [inline cmd/server/customers.go]: Actually, this should add an unblock record, so we can have a way to whitelist a customer.

File: cmd/server/customers.go [hunk 2]
```diff
@@ -21,29 +24,48 @@ var (
 	errNoAuthToken  = errors.New("no authToken provided for webhook")
 	errNoCustomerId = errors.New("no customerId found")
 	errNoNameParam  = errors.New("no name parameter found")
+	errNoUserId     = errors.New("no userId (X-User-Id header) found")
```
Comment (wadearnold) [inline cmd/server/customers.go]: I like that you are following the new error format style

File: cmd/server/customers.go [hunk 3]
```diff
@@ -21,29 +24,48 @@ var (
...
 	Alts      []*ofac.AlternateIdentity `json:"alts"`
 	Comments  []*ofac.SDNComments       `json:"comments"`
+	// Metadata
+	Status *CustomerStatus `json:"status"`
+}
+
+type CustomerBlockStatus string
+
+const (
+	Unsafe    CustomerBlockStatus = "unsafe"
```
Comment (wadearnold) [inline cmd/server/customers.go]: Can we add comments for what these static values represent

File: openapi.yaml [hunk 4]
```diff
@@ -413,7 +413,7 @@ components:
           type: string
           enum:
             - Blocked
-            - Default
+            - Unblocked
```
Comment (wadearnold) [inline openapi.yaml]: Do we need to validate these enums when they are submitted to the service?

Comment (adamdecaf) [reply openapi.yaml]: Yea, but this file is wrong anyway.

**Pull request merged**